packages/core/src/components/card/card.scss

```
@@ -122,3 +122,11 @@
     --ix-card-border-color: var(--theme-card-#{$variant}--border-color);
   }
 }
+
+:host(:focus-visible) {
+  outline: var(--theme-focus-border-thickness) solid
+    var(--theme-color-focus-bdr);
+
+  outline-offset: var(--theme-focus-outline-offset);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)